Fundamentals/Setup/Server-Setup/Load-Balancing/azure-web-apps-v9.md

```diff
@@ -0,0 +1,81 @@
+---
+versionFrom: 9.0.0
+versionTo: 9.4.0
+---
+
+# Load Balancing Azure Web Apps
+
+Ensure you read the [Load Balancing overview](index.md) and general [Azure Web Apps](../azure-web-apps.md) documentation before you begin - you will need to ensure that your ASP.NET Core & logging configurations are correct.
+
+## Azure Requirements
+
+* You will need to setup 2 x Azure Web Apps - one for the backoffice (Administrative) environment and another for your scalable public facing environment (Public)
+* You will need 1 x SQL server that is shared with these 2 web apps
+
+## Lucene/Examine configuration
+
+The single instance Backoffice Administrative Web App should be set to use [SyncedTempFileSystemDirectoryFactory](file-system-replication.md#examine-directory-factory-options).
+
+The multi instance Scalable Public Web App should be set to use [TempFileSystemDirectoryFactory](file-system-replication.md#examine-directory-factory-options).
+
+## Umbraco TEMP files
+
+When an instance of Umbraco starts up it generates some 'temporary' files on disk... in a normal IIS environment these would be created within the folders of the Web Application. In an Azure Web App we want these to be created in the local storage of the actual server that Azure happens to be using for the Web App. So we set this configuration setting to 'true' and the temporary files will be located in the environment temporary folder. This is required for both the performance of the website as well as to prevent file locks from occurring due to the nature of Azure Web Apps shared files system.
```

Comment:
```suggestion
When an instance of Umbraco starts up it generates some 'temporary' files on disk... in a normal IIS environment, these would be created within the folders of the Web Application. In an Azure Web App, we want these to be created in the local storage of the actual server that Azure happens to be used for the Web App. So we set this configuration setting to 'true' and the temporary files will be located in the environment temporary folder. This is required for both the performance of the website as well as to prevent file locks from occurring due to the nature of Azure Web Apps shared files system.
```